projects/1.16/assets/minecraft-earth-mod/minecraft_earth_mod/lang/zh_cn.json

```
@@ -1 +1,190 @@
+  "item.minecraft_earth_mod.muddy_pig_spawn_egg": "泥猪刷怪蛋",
+  "item.minecraft_earth_mod.muddy_pig": "泥猪刷怪蛋",
+  "item.minecraft_earth_mod.mud_formula": "泥浆瓶",
+  "item.minecraft_earth_mod.mud_bucket": "泥浆桶",
+  "item.minecraft_earth_mod.mottled_pig_spawn_egg": "杂斑猪刷怪蛋",
+  "item.minecraft_earth_mod.moolip_spawn_egg": "哞金香刷怪蛋",
+  "item.minecraft_earth_mod.moobloom_spawn_egg": "哞花刷怪蛋",
+  "item.minecraft_earth_mod.moobloom": "哞花刷怪蛋",
+  "item.minecraft_earth_mod.mobof_me_item": "我的生物",
+  "item.minecraft_earth_mod.mob_of_me_spawn_egg": "我的生物刷怪蛋",
+  "item.minecraft_earth_mod.minecraft_earth_dimension": "建造模板之钥",
+  "item.minecraft_earth_mod.midnight_chicken_spawn_egg": "午夜鸡刷怪蛋",
+  "item.minecraft_earth_mod.midnight_chicken": "午夜鸡刷怪蛋",
+  "item.minecraft_earth_mod.merl_spawn_egg": "Merl刷怪蛋",
+  "item.minecraft_earth_mod.melon_golem_spawn_egg": "西瓜傀儡刷怪蛋",
+  "item.minecraft_earth_mod.melon_golem": "西瓜傀儡刷怪蛋",
+  "item.minecraft_earth_mod.lobber_zombie_spawn_egg": "投手僵尸刷怪蛋",
+  "item.minecraft_earth_mod.jolly_llama_spawn_egg": "快乐羊驼刷怪蛋",
+  "item.minecraft_earth_mod.jolly_llama": "快乐羊驼刷怪蛋",
+  "item.minecraft_earth_mod.horned_sheep_spawn_egg": "角羊刷怪蛋",
+  "item.minecraft_earth_mod.horned_sheep": "角羊刷怪蛋",
+  "item.minecraft_earth_mod.horn": "角",
+  "item.minecraft_earth_mod.gold_crested_chicken_spawn_egg": "金冠鸡刷怪蛋",
+  "item.minecraft_earth_mod.glowing_squid_spawn_egg": "发光鱿鱼（测试）刷怪蛋",
+  "item.minecraft_earth_mod.glowing_squid": "发光鱿鱼（测试）刷怪蛋",
+  "item.minecraft_earth_mod.furnace_golem_spawn_egg": "熔炉傀儡刷怪蛋",
+  "item.minecraft_earth_mod.furnace_golem": "熔炉傀儡刷怪蛋",
+  "item.minecraft_earth_mod.fancy_feather": "华丽羽毛",
+  "item.minecraft_earth_mod.fancy_chicken_spawn_egg": "华丽鸡刷怪蛋",
+  "item.minecraft_earth_mod.dyed_cat_spawn_egg": "染色猫刷怪蛋",
+  "item.minecraft_earth_mod.dyed_cat": "染色猫刷怪蛋",
+  "item.minecraft_earth_mod.dried_muddy_pig_spawn_egg": "干燥的泥猪刷怪蛋",
+  "item.minecraft_earth_mod.cookie_cow_spawn_egg": "曲奇牛刷怪蛋",
+  "item.minecraft_earth_mod.convertible_mud": "可转化的泥浆",
+  "item.minecraft_earth_mod.cluckshroom_spawn_egg": "咯菇刷怪蛋",
+  "item.minecraft_earth_mod.cluckshroom": "咯菇刷怪蛋",
+  "item.minecraft_earth_mod.bucket_of_mud": "泥浆桶",
+  "item.minecraft_earth_mod.brule_cow_spawn_egg": "奶油牛刷怪蛋",
+  "item.minecraft_earth_mod.bronzed_chicken_spawn_egg": "青铜鸡刷怪蛋",
+  "item.minecraft_earth_mod.boulder_zombie_spawn_egg": "攀岩僵尸刷怪蛋",
+  "item.minecraft_earth_mod.boneshard": "骨头碎片",
+  "item.minecraft_earth_mod.bone_spider_spawn_egg": "骨蜘蛛刷怪蛋",
+  "item.minecraft_earth_mod.bone_spider": "骨蜘蛛刷怪蛋",
+  "item.minecraft_earth_mod.ashen_cow_spawn_egg": "灰白牛刷怪蛋",
+  "item.minecraft_earth_mod.ashen_cow": "灰白牛刷怪蛋",
+  "item.minecraft_earth_mod.amber_chicken_spawn_egg": "琥珀鸡刷怪蛋",
+  "item.minecraft_earth_mod.amber_chicken": "琥珀鸡刷怪蛋",
+  "item.minecraft_earth_mod.albino_cow_spawn_egg": "白化牛刷怪蛋",
+  "item.minecraft_earth_mod.albino_cow": "白化牛刷怪蛋",
+  "item.minecraft_earth_mobs.muddy_pig": "泥猪刷怪蛋",
+  "gamerule.waterproofGolems.description": "熔炉傀儡在水中会受到伤害吗？",
```

```suggestion
  "gamerule.waterproofGolems.description": "熔炉傀儡在水中是否会受到伤害？",
```
具体应结合实际游戏内容及显示；就键名来看这是一条gamerule的提示